Sources/Classes/RSPreferenceManager.m

```diff
@@ -18,7 +18,8 @@ @implementation RSPreferenceManager
 NSString *const RSServerConfigKey = @"rl_server_config";
 NSString *const RSServerLastUpdatedKey = @"rl_server_last_updated";
 NSString *const RSTraitsKey = @"rl_traits";
-NSString *const RSApplicationInfoKey = @"rl_application_info_key";
+NSString *const RSApplicationBuildKey = @"rl_application_build_key";
```

Comment:
Don't change key.

Reply:
Addressed the concern, by adding the support to migrate the values of `rl_application_info_key` to `rl_application_build_key` if they exist